src/Input/Checkbox/Checkbox.tsx

```diff
@@ -36,13 +37,16 @@ const Checkbox: React.FunctionComponent<CheckboxProps> = ({
       size={size}
       border={border}
       checked={combinedChecked}
+      disabled={disabled}
     >
       <input
         type="checkbox"
         id={id}
         value={value}
-        onClick={handleClick}
+        // onClick={handleClick}
```

Comment:
Do you think we can make Checkbox backward compatible for `onClick` event?  `onClick={onClick}`

And show the warning for it like https://github.com/glints-dev/glints-aries/blob/v4/src/General/Button/Button.tsx#L23 to let the developers gradually replace the onClick event.

Reply:
I tried this but the problem is that the `Change` and `Click` events are not compatible. So we'd have to convert them somehow. I think it may be easier to migrate the few places where this is being used instead.